src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs

```diff
@@ -1050,5 +1089,23 @@ private static unsafe nint UnalignedCountVector128(ref char searchSpace)
             // isn't too important to pin to maintain the alignment.
             return (nint)(uint)(-(int)Unsafe.AsPointer(ref searchSpace) / ElementsPerByte) & (Vector128<ushort>.Count - 1);
         }
+
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool TryFindFirstMatchedLane(Vector128<byte> compareResult, ref int matchedLane)
```

Comment:
I would type `compareResult` as `Vector128<ushort>` to emphasize that it consists of two-byte elements, each of which is either `ffff` or `0000`.  Otherwise, it is not clear what the 'lane' is.